Documentation/project-docs/how-to-track-changes.md

```diff
@@ -0,0 +1,164 @@
+# How to correlate commits with builds
+
+Recently I checked in a change to one of our various dotnet repos (coreclr) and 
+I wanted to figure out when the change would appear in our various NuGet 
+packages and installers. I didn't know how the changes flowed through our 
+build, setup, and publication processes so I asked around (thanks @Dagood!) and 
```

Comment:
It seems kinda silly, but `@Dagood` is a misspelling to my eyes vs. `@dagood`.

Reply:
sure thing : )